File: BlackBerry/AlertMsgs/BuzzCmd.java
package com.mlhsoftware.Blurts;

import org.json.me.*;


class BuzzCmd extends CmdBase 
{
  // Tag Names
  private static final String KEY_BUZZ_LENGTH = "BuzzLength";


  public BuzzCmd( String string ) throws JSONException
  {
    super( string );
  }

  public BuzzCmd()
  {
    super( CMD_BUZZ );
  }

  public void setBuzzLength( int value )
  {
    try
    {
      put( KEY_BUZZ_LENGTH, value );
    }
    catch ( JSONException e )
    {
      System.out.println( e.toString() );
    }
  }

  int getBuzzLength()
  {
    return optInt( KEY_BUZZ_LENGTH );
  }
}

File: Desktop/Domain/CmdMsgs/BuzzCmd.cs
using System;

using System.Xml;
using System.Xml.Xsl;
using System.Collections;
using System.Runtime.InteropServices;

namespace Blurts
{
  [ComVisible(true)]
  public class BuzzCmd : CmdBase
  {
    // Tag Names
    private static String KEY_BUZZ_LENGTH = "BuzzLength";


    public BuzzCmd(String s)
      : base(s)
    {
    }

    public BuzzCmd()
      : base(CMD_BUZZ)
    {
      put(KEY_BUZZ_LENGTH, 600);
    }

    [ComVisible(true)]
    public int BuzzLength
    {
      get
      {
        return optInt(KEY_BUZZ_LENGTH);
      }
      set
      {
        try
        {
          remove(KEY_BUZZ_LENGTH);
          put(KEY_BUZZ_LENGTH, value);
        }
        catch (JSONException e)
        {
          Console.WriteLine(e.ToString());
        }
      }
    }
  }
}
